### PR TITLE
XW-4192 | subtitles/captions in featured videos

### PR DESCRIPTION
**Pull request by latata**
@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 1]
```diff
@@ -80,9 +84,13 @@ require([
...
 			},
 			autoplay: willAutoplay,
+			captions: {
+				enabled: featuredVideoCookieService.areCaptionsEnabled()
+			},
 			settings: {
 				showAutoplayToggle: true,
-				showQuality: true
+				showQuality: true,
+				showCaptionsToggle: true
```
Comment (hakubo) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js:93]: it won't be a toogle soon, should we rename it already?

File: extensions/wikia/ArticleVideo/scripts/featured-video.cookies.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		return function () {
+			return cookies.get(cookieName) !== '0';
+		};
+	}
+
+	return {
+		isAutoplayEnabled: isCookieEnabled(autoplayCookieName),
+		areCaptionsEnabled: isCookieEnabled(captionsCookieName),
+		toggleAutoplay: toggleCookie(autoplayCookieName),
+		toggleCaptions: toggleCookie(captionsCookieName)
```
Comment (hakubo) [inline extensions/wikia/ArticleVideo/scripts/featured-video.cookies.js:34]: it supports only on/off, should it support, what language was chosen?

**Review by latata — COMMENTED** [review]
LGTM

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js (hunk 1 above)
Comment (rybmat) [reply extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js:93]: I would rename it when we'll be doing the other solution

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/ArticleVideo/scripts/featured-video.cookies.js (hunk 2 above)
Comment (xkxd) [reply extensions/wikia/ArticleVideo/scripts/featured-video.cookies.js:34]: I thought about taking that into account, but I'd agree with @rybmat to not include this here but in specific feature / PR

**Pull request merged**